contribs/shared_mobility/src/main/java/org/matsim/contrib/shared_mobility/run/SharingServiceConfigGroup.java

```diff
@@ -4,14 +4,15 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import javax.annotation.Nonnegative;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
 
 import org.matsim.core.config.Config;
 import org.matsim.core.config.ReflectiveConfigGroup;
 import org.matsim.core.config.consistency.BeanValidationConfigConsistencyChecker;
 
+import javax.annotation.Nonnegative;
```

Comment:
shouldn't this also be `jakarta.*`?

Reply:
The Nonnegative annotation is only existing in the jsr305 dependecy from google. Jakarta does not support this annotation.